### PR TITLE
Update changes-ee-v5.md with e5.7.2 changes

### PR DESCRIPTION
**Pull request by yanzhiemq**

### CODE REVIEW

File: en_US/changes/changes-ee-v5.md [hunk 1]
```diff
@@ -1,5 +1,121 @@
...
+- [#13386](https://github.com/emqx/emqx/pull/13386) Added support for initializing a list of banned clients on an empty EMQX node or cluster with a bootstrap file in CSV format. The corresponding config entry to specify the file path is `banned.bootstrap_file`. This file is a CSV file with `,` as its delimiter. The first line of this file must be a header line. All valid headers are listed here:
+
+  - as :: required
+  - who :: required
+  - by  :: optional
+  - reason :: optional
+  - at :: optional
+  - until :: optional
+  
+  See the [Configuration Manual](https://docs.emqx.com/en/enterprise/v@EE_VERSION@/hocon/) for details on each field.
```
Comment (hjianbo) [inline en_US/changes/changes-ee-v5.md:38]: Invalid link

File: en_US/changes/changes-ee-v5.md [hunk 2]
```diff
@@ -1,5 +1,121 @@
...
+  
+  Each row in the rest of this file must contain the same number of columns as the header line, and the column can be omitted then its value is `undefined`.
+  
+- [#13452](https://github.com/emqx/emqx/pull/13452) Kafka producer action's `topic` configuration now supports templates.
+
+  Ensure that topics are pre-existing in Kafka. If a message is directed to a non-existent topic (given Kafka's disabled topic auto-creation), the message will fail with an unrecoverable error. Additionally, if a message lacks sufficient information to match the configured template, it will also fail with an unrecoverable error. For example, the template is `t-${t}` but the message context lacks a `t` definition. For detailed information, see [Kafka Dynamic Topics](../data-integration/data-bridge-kafka.md#kafka-dynamic-topics).
```
Comment (hjianbo) [inline en_US/changes/changes-ee-v5.md]: invalid link for Kafka dynamic topics

File: en_US/changes/changes-ee-v5.md (hunk 1 above)
Comment (id) [reply en_US/changes/changes-ee-v5.md:38]: `@EE_VERSION` will be replaced with 5.7.2 when rendering.

File: en_US/changes/changes-ee-v5.md (hunk 2 above)
Comment (Meggielqk) [reply en_US/changes/changes-ee-v5.md]: I will update the link when the relevant PR is merged.

File: en_US/changes/changes-ee-v5.md [hunk 3]
```diff
@@ -1,5 +1,121 @@
 # Version 5
 
+## 5.7.2
+
+*Release Date: 2024-08-07*
+
+### Enhancements
+
+- [#13317](https://github.com/emqx/emqx/pull/13317) Added a new per-authorization source metric type: `ignore`.  This metric increments when an authorization source attempts to authorize a request but encounters scenarios where the authorization is not applicable or encounters an error, resulting in an undecidable outcome.
```
Comment (Meggielqk) [inline en_US/changes/changes-ee-v5.md]:
```suggestion
- [#13317](https://github.com/emqx/emqx/pull/13317) Added a new per-authorization source metric type: `ignore`.  This metric increments when an authorization source attempts to authorize a request but encounters scenarios where the authorizer is not applicable or encounters an error, resulting in an undecidable outcome.
```

File: en_US/changes/breaking-changes-ee-5.7.md [hunk 4]
```diff
@@ -1,5 +1,12 @@
 # Incompatible Changes in EMQX 5.7
 
+
+## e5.7.2
+
+- [#13327](https://github.com/emqx/emqx/pull/13327) The fix for issues in Kafka, Confluent, and Azure Event Hubs integrations has resulted in a change to the directory path scheme for on-disk buffers of these integrations. The new scheme uses the action name instead of the topic name. Upgrading to this version will invalidate old buffer files, and require manual cleanup of the old directories.
+- [#13332](https://github.com/emqx/emqx/pull/13332) Error messages for misconfigured Amazon S3 integrations are now more descriptive and user-friendly. The Amazon S3 Sink configurations with an invalid object key templates for aggregated upload will no longer work. Before this change, such configurations were considered valid but but resulted in non-functional integrations.
```
Comment (id) [inline en_US/changes/breaking-changes-ee-5.7.md]:
```suggestion
- [#13332](https://github.com/emqx/emqx/pull/13332) Error messages for misconfigured Amazon S3 integrations are now more descriptive and user-friendly. The Amazon S3 Sink configurations with an invalid object key templates for aggregated upload will no longer work. Before this change, such configurations were considered valid but resulted in non-functional integrations.
```

File: en_US/changes/changes-ee-v5.md [hunk 5]
```diff
@@ -1,5 +1,121 @@
...
+
+- [#13541](https://github.com/emqx/emqx/pull/13541) Fixed an issue where disabling CRL checks for a listener required a listener restart to take effect.
+
+- [#13305](https://github.com/emqx/emqx/pull/13305) Improved error handling for Redis connectors. Previously, Redis connectors with Redis Mode set as `single` or `sentinel` would always encounter a timeout error during the connector test in the Dashboard if no username or password was provided. This update ensures that users now receive an informative error message in such scenarios. Additionally, more detailed error information has been added for all Redis connector types to enhance diagnostics and troubleshooting.
+
+- [#13327](https://github.com/emqx/emqx/pull/13327) Fixed an issue in Kafka, Confluent, and Azure Event Hub integrations where multiple actions reusing the same connector and configured with the same topic could interfere with each other when one of the actions was deleted or disabled. For example, data writing of other actions might be affected.
```
Comment (id) [inline en_US/changes/changes-ee-v5.md]:
```suggestion
- [#13327](https://github.com/emqx/emqx/pull/13327) Fixed an issue in Kafka, Confluent, and Azure Event Hubs integrations where multiple actions reusing the same connector and configured with the same topic could interfere with each other when one of the actions was deleted or disabled. For example, data writing of other actions might be affected.
```

**Review by id — DISMISSED** [review]

**Review by id — DISMISSED** [review]

File: zh_CN/changes/changes-ee-v5.md [hunk 6]
```diff
@@ -1,5 +1,91 @@
...
+- [#13422](https://github.com/emqx/emqx/pull/13422) 解决了无法将选项 `force_shutdown.max_heap_size` 设置为 0 以禁用此调试的问题。
+- [#13442](https://github.com/emqx/emqx/pull/13442) 解决了动作和 Source 的健康检查间隔配置未被遵守的问题。之前，EMQX 忽略了为动作指定的健康检查间隔，并使用连接器的间隔。此修复确保 EMQX 现在正确使用为动作/ Source 配置的健康检查间隔，允许独立和准确的健康监控频率。
+- [#13503](https://github.com/emqx/emqx/pull/13503) 修复了连接器在初始启动时未遵循配置的健康检查间隔的问题，需要更新或重启以应用正确的间隔。
+- [#13515](https://github.com/emqx/emqx/pull/13515) 修复了当节点因某些原因宕机时，同一客户端无法订阅相同的排它主题的问题。
+- [#13527](https://github.com/emqx/emqx/pull/13527) 在规则引擎中修复了一个问题，即当 `$bridges/...` 被包含在 `FROM` 子句中时，执行消息发布事件的 SQL 测试始终返回空结果。
+- [#13541](https://github.com/emqx/emqx/pull/13541) 修复了禁用监听器的 CRL 检查需要监听器重启才能生效的问题。
+- [#13305](https://github.com/emqx/emqx/pull/13305) 改进了 Redis 连接器的错误处理。之前，如果 Redis 连接器的 Redis 模式设置为 `single` 或 `sentinel`，且未提供用户名或密码，那么在 Dashboard 中进行连接器测试时会始终遇到连接超时错误。此更新确保用户现在能在这种情况下收到详细的错误信息。此外，还为所有 Redis 连接器类型添加了更详细的错误信息，以增强诊断和故障排除能力。
+- [#13327](https://github.com/emqx/emqx/pull/13327) 修复了 Kafka、Confluent 和 Azure Event Hub 集成中的问题，如果多个动作复用同一个 connector 并配置了相同的主题，当删除或禁用其中一个动作时可能会相互干扰，比如影响其他动作的数据写入。
```
Comment (Meggielqk) [inline zh_CN/changes/changes-ee-v5.md]:
```suggestion
- [#13327](https://github.com/emqx/emqx/pull/13327) 修复了 Kafka、Confluent 和 Azure Event Hubs 集成中的问题，如果多个动作复用同一个 connector 并配置了相同的主题，当删除或禁用其中一个动作时可能会相互干扰，比如影响其他动作的数据写入。
```

File: zh_CN/changes/changes-ce-v5.md [hunk 7]
```diff
@@ -1,5 +1,71 @@
 # v5 版本
 
+## 5.7.2
+
+*发布日期：2024-08-07*
+
+### 增强
+
+- [#13317](https://github.com/emqx/emqx/pull/13317) 增加了一种新的针对授权源的指标类型：`ignore`。当授权源尝试对请求进行授权但遇到不适用或出现错误导致无法决定结果的情况时，此指标计数将递增。
```
Comment (Meggielqk) [inline zh_CN/changes/changes-ce-v5.md]:
```suggestion
- [#13317](https://github.com/emqx/emqx/pull/13317) 增加了一种新的针对授权源的指标类型：`ignore`。当授权源尝试对请求进行授权但遇到授权源不适用或出现错误导致无法决定结果的情况时，此指标计数将递增。

```

File: en_US/changes/changes-ce-v5.md [hunk 8]
```diff
@@ -1,5 +1,93 @@
 # Version 5
 
+## 5.7.2
+
+*Release Date: 2024-08-07*
+
+### Enhancements
+
+- [#13317](https://github.com/emqx/emqx/pull/13317) Added a new per-authorization source metric type: `ignore`.  This metric increments when an authorization source attempts to authorize a request but encounters scenarios where the authorization is not applicable or encounters an error, resulting in an undecidable outcome.
```
Comment (Meggielqk) [inline en_US/changes/changes-ce-v5.md]:
```suggestion
- [#13317](https://github.com/emqx/emqx/pull/13317) Added a new per-authorization source metric type: `ignore`.  This metric increments when an authorization source attempts to authorize a request but encounters scenarios where the authorizer is not applicable or encounters an error, resulting in an undecidable outcome.

```

**Review by id — APPROVED** [review]

**Pull request merged**